webpack_config/devServerHTTPS.js

```diff
@@ -0,0 +1 @@
+npm run dev:https
```

Comment:
What is this file for?

Reply:
@dternyak I'm not sure... ignore it if it makes no sense for you (I guess I might have added it long time ago trying to run the site as https and I kept it by mistake)

Reply:
Got it. Do you mind removing this file in that case?

Reply:
Done